### DISCUSSION TITLE
About `create space`, the sotrage leader can not distribute more balanced

### DISCUSSION THREAD

**Discussion by Milittle** [vesoft-inc/nebula#4731] [Ideas]
When we create space, we meet many times, if we specify the replicator is 3, the space partition's leader distribute in two storage, the third sotrage can not ditribute leader.

I think this problem caused by the storage's metaclient period method coming time.

Guys, do you have some good strategy to avoid phenomenon in initially creating space？ or we can discuss for this.

:) I already know about `balance leader` can re-distribute the leader's distribution.

Thanks, anyway.

**Comment by wey-gu** [comment]
see also discussion in https://discuss.nebula-graph.com.cn/t/topic/10720/4
